<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书馆书籍管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addBookButton">
        <property name="text">
         <string>添加图书</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeBookButton">
        <property name="text">
         <string>删除图书</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchBookButton">
        <property name="text">
         <string>查询图书</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showAllBooksButton">
        <property name="text">
         <string>显示所有图书</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateBookStatusButton">
        <property name="text">
         <string>更新借阅状态</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="allBooksTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
